src/components/editor/property/InputListLOC.jsx

```diff
@@ -145,13 +145,16 @@ const mapStateToProps = (state, ownProps) => {
   const errors = findErrors(state.selectorReducer, ownProps.reduxPath)
 
   // Make sure that every item has a label
+  // TODO: Move this to the ResourceStateBuilder
```

Comment:
Could you create a ticket for moving this to the ResourceStateBuilder and reference the issue here?

Reply:
https://github.com/LD4P/sinopia_editor/issues/1117